WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/ProductNameGeneration/ProductNameGenerationView.swift

```diff
@@ -98,33 +101,33 @@ struct ProductNameGenerationView: View {
 
             Spacer()
 
-            Divider()
-                .renderedIf(viewModel.generationInProgress == false && viewModel.hasGeneratedMessage)
-
-            HStack(spacing: Constants.horizontalSpacing) {
-                // Action button to regenerate product name
-                Button(action: {
-                    viewModel.generateProductName()
-                }, label: {
-                    Label {
-                        Text(viewModel.generateButtonTitle)
-                    } icon: {
-                        Image(uiImage: viewModel.generateButtonImage)
-                            .renderingMode(.template)
+            if let suggestedText = viewModel.suggestedText,
+               viewModel.generationInProgress == false {
+                Divider()
```

Comment:
super nit: I think that adding a line space will improve readability.

Reply:
Done in 0525836 👍